migrations/V20__Add_Initial_Submitted_At_Biz_Case.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE business_case ADD COLUMN initial_submitted_at timestamp with time zone;
+ALTER TABLE business_case ADD COLUMN last_submitted_at timestamp with time zone;
+-- The update likely doesn't do anything because we weren't updating submitted at date
+-- in the storage package.
+UPDATE business_case
+    SET initial_submitted_at = submitted_at, last_submitted_at = submitted_at
```

Comment:
You mentioned two PRs at some point in a standup, think? I'm guessing you'll drop the old column once this work is live? That's a good practice, just want to make sure we're on the same page.

Reply:
yep. That's the plan